docs/core/diagnostics/metrics-instrumentation.md

```diff
@@ -198,18 +254,18 @@ using System.Threading;
 
 class Program
 {
-    static Meter s_meter = new Meter("HatCo.HatStore", "1.0.0");
-    static Counter<int> s_hatsSold = s_meter.CreateCounter<int>("hats-sold");
-    static Histogram<int> s_orderProcessingTimeMs = s_meter.CreateHistogram<int>("order-processing-time");
+    static Meter s_meter = new Meter("HatCo.Store");
+    static Counter<int> s_hatsSold = s_meter.CreateCounter<int>("hatco.store.hats_sold");
+    static Histogram<int> s_orderProcessingTimeMs = s_meter.CreateHistogram<int>("hatco.store.order_processing_time");
```

Comment:
Change to seconds? Change type to long? Also, add "s" as unit

Reply:
I see this in recommendations:

> For timing things, Histogram is usually preferred. Often it's useful to understand the tail of these distributions (90th, 95th, 99th percentile) rather than averages or totals.

Recommend seconds?

Reply:
I changed it to seconds, but didn't include seconds in the unit because we haven't reached a discussion of units in the doc yet.